src/gadgets/mpt_update.rs

```diff
@@ -137,6 +137,7 @@ impl MptUpdateConfig {
         key_bit: &impl KeyBitLookup,
         rlc: &impl RlcLookup,
         bytes: &impl BytesLookup,
+        rlc_randomness: &RlcRandomness,
```

Comment:
in line 145:  "let [old_value, new_value] = cb.advice_columns(cs);" should be changed to "let [old_value, new_value] = cb.advice_columns_in_phase(cs, Phase2); ?